Context.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
```

Comment:
let's have lowercase filenames

Reply:
if the file consists of class then the filename should be uppercase.